index.html

```diff
@@ -4,8 +4,36 @@
     <meta charset="UTF-8">
     <title>Rocket Man</title>
     <link rel="stylesheet" href="css/game.css" type="text/css">
+    <link rel="stylesheet" href="css/styles.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Play:400,700" rel="stylesheet">
 </head>
 <body>
+    <div id="splash-screen">
+        <div id="splash-title-wrapper">
+            <h1 id="splash-title">ROCKET MAN</h1>
+            <div id="splash-title-rocket">
+                <img src="img/fire-png-gif-489.gif" id="splash-flames">
+                <img src="img/rocket-large.png" id="splash-rocket-img">
+            </div>
+        </div>
+        <div id="splash-open-game">Play Me</div>
+        <div id="splash-bottom-wrapper">
+            <img src="img/new-york-large.png" id="splash-city">
+            <div id="splash-instructions">
```

Comment:
the instructions fall off the bottom of the box in firefox

Reply:
cannot reproduce